libfreerdp/core/orders.c

```diff
@@ -2714,6 +2714,8 @@ static CACHE_BRUSH_ORDER* update_read_cache_brush_order(rdpUpdate* update, wStre
 
 	Stream_Read_UINT8(s, cache_brush->cx);     /* cx (1 byte) */
 	Stream_Read_UINT8(s, cache_brush->cy);     /* cy (1 byte) */
+	/* according to  Section 2.2.2.2.1.2.7 errata the windows implementation sets this filed is set
+	 * to 0x00 */
```

Comment:
"sets this filed is set to 0x00" --> "sets this **field** to 0x00"

Reply:
this is a bug that is also on master, fix there. here we only do backports